src/Pages/Services/PageLayout/index.tsx

```diff
@@ -7,17 +8,22 @@ import {
 } from "./style";
 import titlebBaseImage from "../../../assets/images/title-base.png";
 import { PageCarousel } from "./PageCarousel";
+import React from "react";
 
 interface ServiceDescriptionProps {
   title: string;
   content: string[];
   imageUrl: { id: number; image: string }[];
+  contactNumber: { id: number; number: string }[];
+  emailId: string[];
```

Comment:
```suggestion
  emailId?: string[];
```

And then you will not need to pass emailId if not required